Makefile

```diff
@@ -53,6 +53,8 @@ grpc-extension: .pre-build
 # Convenience tools
 osqueryi-tables: table.ext
 	osqueryd -S --allow-unsafe --verbose --extension ./build/darwin/tables.ext
+osqueryi-tables-windows: table.ext
+	C:\Program Files\osquery\osqueryd\osqueryd.exe -S --allow-unsafe --verbose --extension .\build\windows\tables.exe
```

Comment:
Hardcoded this path because it's where osquery is normally installed on Windows using the default package...and the package doesn't add it to the path. Felt ok for our purposes...

Reply:
Sure np.